setup/setup_dev.py

```diff
@@ -117,7 +117,7 @@ def cmd_setup_git_review():
 def setup_deps_debian(git_dir):
     debian_control_path = os.path.join(git_dir, 'debian/control')
     debian_control = open(debian_control_path).read()
-    debs = re.findall('python-[0-9a-z]+',debian_control)
+    debs = re.findall('python-3[0-9a-z]+',debian_control)
```

Comment:
Maybe it should be `'python3-[...]` instead of `'python-3[...]`, :thinking: ?

Reply:
you're right, annoying typo. Thanks